tasks/GiveMeAHand.tex

```diff
@@ -27,38 +27,44 @@ \subsection*{Setup}
+		\item Delivery instructions may include phrases like: \textit{"Place this apple on that plate"} or \textit{"Put this can in the trash bin."} The operator gives instructions using both verbal commands and gestures (e.g., \textit{"Place this item there} \textit{(pointing), next to the flower vase"}).
+		\item At any moment, the robot may ask questions to confirm information, request clarification, or ask the operator to repeat the instruction.
+		\item Since the target locations are intentionally ambiguous, the robot must request clarification from the operator before proceeding. This must be done through natural interaction, such as verbal questions or multimodal cues (e.g., pointing, asking “Which one?”). The robot must not guess or place the object without confirmation.
+		\item Full points for a correct delivery will only be awarded if the object is placed inside the designated container (e.g., bins) or within a few centimeters of the intended location (e.g., plates or surfaces).
+		\item In this task, natural human-robot interaction is the primary focus. Before engaging with the operator to request clarification, the robot must first capture their attention — for example, through visual signals, gestures, or sound cues — rather than relying on unattended interactions (e.g., talking without addressing a specific person). This is particularly important when the operator and the robot are not in close proximity; for instance, the operator might be occupied retrieving the next object or speaking with the referee.
 	\end{itemize}
-	\item \textbf{Object manipulation:} 
-        \begin{itemize}[nosep]
-            \item While task-action instructions during handing over objects like \textit{Please, place the item in my gripper and wait until I close it in three, two, one} will be considered \textit{Deus ex Machina}, the robot may warn the user about closing their manipulator, for example, \textit{I have reached the object, I will close my gripper in three seconds -- please, be careful}.
-            \item The goal locations of objects in the environment may introduce some ambiguity. For example, there could be three bins or two plates positioned close to one another. Delivery instructions may specify actions like: \textit{"Place this apple on that plate"} or \textit{"Put this can in the trash bin."}
-            \item Full points for a correct delivery will only be awarded if the object is placed inside (bins) or within a few centimeters (on surfaces) of the designated target location. The robot may seek confirmation or request further clarification if needed (e.g., Robot: \textit{"Is this the correct location?"} User: \textit{"No, move the object to the bin on your left."}).
-        \end{itemize}
 \end{enumerate}
 
 \subsection*{Instructions:}
 \subsubsection*{Referee instructions}
 
+Timer starts when the operator begins calling to the robot.
```

Comment:
I always find that starting inside the arena makes it more chaotic. Is there a rationale for not just starting outside?

Reply:
Good point. It can start outside the arena and define the room two hours before the test

Reply:
Yes, I prefer tasks starting at outside the entrance